pyproject.toml

```diff
@@ -69,6 +69,7 @@ jpeg2000 = [
   "lxml>=4.9.0,!=5.0.0",
 ]
 map = [
+  "contourpy>=1.2.0",
```

Comment:
```suggestion
```
I thought this was a dependency of MPL?

Reply:
Yes, understood now as mpl already has `ContourPy` as its dependency then we don't need to add it as well